core/models.py

```diff
@@ -260,3 +260,16 @@ def save(self, *args, **kwargs):
 
     def __str__(self):
         return str(self.name)
+
+
+class Subscriber(models.Model):
+    subscriber_uuid = models.CharField(primary_key=True, max_length=255, verbose_name='Subscriber UUID', default=uuid.uuid4, unique=True)
+    email = models.EmailField(verbose_name='Subscriber Email', blank=False, null=False)
```

Comment:
Can we also save name here? This can be blank as well and might help in future to send targeted emails